### PR TITLE
Reflect bats changes (https://github.com/exercism/bash/pull/541)

### PR DESCRIPTION
**Pull request by braoult**

### CODE REVIEW

**Comment by iHiD** [comment]
This must continue to work on people's older solutions and newer solutions, so we'll need support for both in here.

**Comment by braoult** [comment]
> This must continue to work on people's older solutions and newer solutions, so we'll need support for both in here.

It does support both current and new file names. This is done line 78 of `run.sh`:
```
# test scripts may be (old) xxxx_test.sh
[[ -f "$test_file" ]] || test_file=${slug//-/_}_test.sh
```

In fact, if you look at `tests` changes, you will notice I renamed 4/7 `bats` files to new names, and kept 3 with old name, exactly for that.

**Review by glennj — APPROVED** [review]
Looks good.

File: bin/run.sh [hunk 1]
```diff
@@ -66,14 +66,17 @@ run_tests() {
...
+    local test_file=${slug//-/_}.bats
     local solution_dir="$2"
     local output_file="$3"
 
     echo "Running tests."
 
     cd "$solution_dir"
 
+    # test scripts may be (old) xxxx_test.sh
+    [[ -f "$test_file" ]] || test_file=${slug//-/_}_test.sh
```
Comment (IsaacG) [inline bin/run.sh]: Would it be cleaner to have the old and new names set adjacent to each other?

Comment (braoult) [reply bin/run.sh]: Not for me, but I did it (in [this commit](https://github.com/exercism/bash-test-runner/pull/27/commits/ced2fd62d4897662e59b8350036332d5d270d9fc)).

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**